framework/db/BaseActiveRecord.php

```diff
@@ -1783,7 +1783,7 @@ private function isValueDifferent($newValue, $oldValue)
     {
         if (is_array($newValue) && is_array($oldValue)) {
             // Only sort associative arrays
-            $sorter = function(&$array) {
+            $sorter = function (&$array) {
```

Comment:
No need to change it.

Reply:
returned the codestyle to its original form